src/libPMacc/include/eventSystem/tasks/TaskSetValue.hpp

```diff
@@ -174,14 +174,17 @@ class TaskSetValue<T_ValueType, T_dim, true> : public TaskSetValueBase<T_ValueTy
     {
         size_t current_size = this->destination->getCurrentSize();
         const DataSpace<dim> area_size(this->destination->getCurrentDataSpace(current_size));
-        dim3 gridSize = area_size;
 
-        /* line wise thread blocks*/
-        gridSize.x = ceil(double(gridSize.x) / 256.);
+        if(area_size.productOfComponents() != 0)
+        {
+            dim3 gridSize = area_size;
 
-        kernelSetValue << <gridSize, 256, 0, this->getCudaStream() >> >
-            (this->destination->getDataBox(), this->value, area_size);
+            /* line wise thread blocks*/
+            gridSize.x = ceil(double(gridSize.x) / 256.);
 
+            kernelSetValue << <gridSize, 256, 0, this->getCudaStream() >> >
```

Comment:
a bit weird bracket scheme, please use `<<<` since we found compilers have trouble with that (clang --cuda?)